docker-compose.yml

```diff
@@ -119,7 +123,8 @@ services:
     network_mode: service:tor
 
   celery-beat:
-    image: backend
+    image: backend-image
+    pull_policy: never
     container_name: cele-beat-dev
     restart: always
     environment:
```

Comment:
This seems to fix the problems in OSx, please @Reckless-Satoshi can you confirm I didn't break other OS?